MiBench/consumer/jpeg/djpeg/src/djpeg.c

```diff
@@ -415,180 +415,143 @@ COM_handler (j_decompress_ptr cinfo)
  * The main program.
  */
 
-int
-main (int argc, char **argv)
-{
+
+int main(int argc, char **argv) {
   struct jpeg_decompress_struct cinfo;
   struct jpeg_error_mgr jerr;
 #ifdef PROGRESS_REPORT
   struct cdjpeg_progress_mgr progress;
 #endif
   int file_index;
   djpeg_dest_ptr dest_mgr = NULL;
-  FILE * input_file;
-  FILE * output_file;
+  FILE *input_file;
+  FILE *output_file;
   JDIMENSION num_scanlines;
 
-  /* On Mac, fetch a command line. */
 #ifdef USE_CCOMMAND
   argc = ccommand(&argv);
 #endif
 
   progname = argv[0];
   if (progname == NULL || progname[0] == 0)
-    progname = "djpeg";		/* in case C library doesn't provide it */
+    progname = "djpeg";
+
+  clock_t start, end;
+  start = clock();
 
-  /* Initialize the JPEG decompression object with default error handling. */
   cinfo.err = jpeg_std_error(&jerr);
   jpeg_create_decompress(&cinfo);
-  /* Add some application-specific error messages (from cderror.h) */
   jerr.addon_message_table = cdjpeg_message_table;
   jerr.first_addon_message = JMSG_FIRSTADDONCODE;
   jerr.last_addon_message = JMSG_LASTADDONCODE;
-  /* Insert custom COM marker processor. */
   jpeg_set_marker_processor(&cinfo, JPEG_COM, COM_handler);
 
-  /* Now safe to enable signal catcher. */
 #ifdef NEED_SIGNAL_CATCHER
   enable_signal_catcher((j_common_ptr) &cinfo);
 #endif
 
-  /* Scan command line to find file names. */
-  /* It is convenient to use just one switch-parsing routine, but the switch
```

Comment:
Do not remove these and other comments from the original code.

Reply:
When I started, I was looking for comments to understand, so for keeping the repo locally I removed the initial headings. Will not do that again, and will modify only in the required places.